service/selectors/entities/channels.js

```diff
@@ -148,3 +149,22 @@ export const getAutocompleteChannelWithSections = createSelector(
         return channels;
     }
 );
+
+export const canManageChannelMembers = createSelector(
+    getCurrentChannel,
+    getCurrentChannelMembership,
+    getCurrentTeamMembership,
+    getUsers,
+    getCurrentUserId,
+    (channel, channelMembership, teamMembership, allUsers, currentUserId) => {
+        const user = allUsers[currentUserId];
+        const roles = `${channelMembership.roles} ${teamMembership.roles} ${user.roles}`;
```

Comment:
@enahum, @jarredwitt Believe me guys, I first tried using `getCurrentUserRoles` here. But because it's a true selector, it can't be used as an input for another selector; only pure functions can be. This was news to me. That's why I had to replicate that selector here. I probably should've explained that in a code comment or a PR comment.
<img width="487" alt="mm-rn--input-selectors-must-be-functions-error" src="https://cloud.githubusercontent.com/assets/3317960/23325971/7cf4412a-faae-11e6-887c-ec1e2ad0967f.png">